bot/exts/evergreen/ping.py

```diff
@@ -1,11 +1,14 @@
+import arrow
+from dateutil.relativedelta import relativedelta
 from discord import Embed
 from discord.ext import commands
 
+from bot import start_time
 from bot.constants import Colours
 
 
 class Ping(commands.Cog):
-    """Ping the bot to see its latency and state."""
+    """Get info about the bot's ping and uptime."""
 
     def __init__(self, bot: commands.Bot):
```

Comment:
In #718, all `commands.Bot` type hints were changed to `Bot` from `bot/bot.py`, we should change the type hint here and in the setup function.

Reply:
I don't see why you'd do that, it's already done [here](https://github.com/python-discord/sir-lancebot/pull/718/files#diff-f325491078a141e8bdd8e16f8c395d06b4d11383e4b9d29876598fad7b1cc68a).

Reply:
Alright, should be good then.